src/viur/core/modules/user.py

```diff
@@ -773,14 +775,14 @@ def otp(self, *args, **kwargs):
         if res is None:
             otp_user_conf["attempts"] = attempts + 1
             session.markChanged()
-
+            skel = self.OtpSkel()
+            skel.errors = [ReadFromClientError(ReadFromClientErrorSeverity.Invalid, "Wrong OTP Token", ["otptoken"])]
```

Comment:
Error shall not be set on first call.
```suggestion
            if attempts > 0:
                skel.errors = [ReadFromClientError(ReadFromClientErrorSeverity.Invalid, "Wrong OTP Token", ["otptoken"])]
```

Reply:
But why ? We start with the `start` method and send the token and the we must send an error back ?

Reply:
OK it seems you're right.